### PR TITLE
fix(login-page): add missing id and adapt label ref

### PR DESCRIPTION
**Pull request by flegastelois**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #NA

### CODE REVIEW

File: templates/pages/login.html.twig [hunk 1]
```diff
@@ -60,27 +60,27 @@
...
                   {% endif %}
                </label>
-               <input type="password" class="form-control" name="{{ pwdfield }}" placeholder="" autocomplete="off" tabindex="2" />
+               <input type="password" class="form-control" id="login_password" name="{{ pwdfield }}" placeholder="" autocomplete="off" tabindex="2" />
             </div>
 
             {% if constant('GLPI_DEMO_MODE') %}
                <div class="mb-3">
-                  <label class="form-label">{{ __('Language') }}</label>
+                  <label class="form-label" for="dropdown_language1">{{ __('Language') }}</label>
```
Comment (trasher) [inline templates/pages/login.html.twig:68]:
```suggestion
                  <label class="form-label" for="dropdown_language{{ rand }}">{{ __('Language') }}</label>
```

File: templates/pages/login.html.twig [hunk 2]
```diff
@@ -60,27 +60,27 @@
...
                   {{ languages_dropdown|raw }}
                </div>
             {% endif %}
 
             {% if config('display_login_source') %}
                <div class="mb-3">
-                  <label class="form-label">{{ __('Login source') }}</label>
+                  <label class="form-label" for="dropdown_auth1">{{ __('Login source') }}</label>
```
Comment (trasher) [inline templates/pages/login.html.twig:75]:
```suggestion
                  <label class="form-label" for="dropdown_auth{{ rand }}">{{ __('Login source') }}</label>
```

File: index.php [hunk 3]
```diff
@@ -149,6 +149,7 @@
                               ]),
         'languages_dropdown'  => Dropdown::showLanguages('language', [
             'display'             => false,
+            'rand'                => '1',
```
Comment (cedric-anne) [inline index.php:152]: Would be better to use `{% set rand = random() %}` in the template.
```suggestion
```

File: templates/pages/login.html.twig (hunk 2 above)
Comment (trasher) [reply templates/pages/login.html.twig:75]: I do not understand :/ Ids are set from `{{ auth_dropdown_login|raw }}`, this already take care of passed `rand` param as far as I know

Comment (trasher) [reply templates/pages/login.html.twig:75]: Ohhh... OK? I understood. With @cedric-anne proposal, this should be ok :)

File: index.php (hunk 3 above)
Comment (trasher) [reply index.php:152]: An therefore pass it to all php dropdown methods calls?

Comment (cedric-anne) [reply index.php:152]: Probably, yes.

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Review by orthagh — APPROVED** [review]

**Pull request merged**